processor/logdedupprocessor/counter.go

```diff
@@ -144,21 +162,32 @@ func (a *logCounter) Increment() {
 	a.count++
 }
 
-// getLogKey creates a unique hash for the log record to use as a map key
-/* #nosec G104 -- According to Hash interface write can never return an error */
-func getLogKey(logRecord plog.LogRecord) [8]byte {
-	hasher := fnv.New64()
-	attrHash := pdatautil.MapHash(logRecord.Attributes())
+// getResourceKey creates a unique hash for the resource to use as a map key
+func getResourceKey(resource pcommon.Resource) uint64 {
+	hasher := xxhash.New()
```

Comment:
What's the rationale for wrapping the pdatautil output? In general I think we should standardize where possible. If there's something deficient about the pdatautil implementation, can we fix or enhance it? Maybe just a `pdatautil.ResourceHash` which aliases `MapHash` so that there's an unambiguous standard?